applications/Simple/README.md

```diff
@@ -10,19 +10,40 @@ License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 either express or implied. 
 See the License for the specific language governing permissions and limitations under the License.
 
-
 Simple
 ======
 This sample is a very simple application, deploying a trivial EPL program into
 a running correlator. A container runs the correlator and then runs the client
 tools to deploys the program.
 
-Detailed instructions on running the sample either as a Compose-based
-application or via Kubernetes can be found in the README in the parent
-directory. If you are running via Kubernetes this sample creates the following
+If you are running via Kubernetes this sample creates the following
 resources which can be accessed via logs and must be deleted via delete:
 
 	* pod simple-sample
 
+Running with Docker-Compose
+==============
+
+Get the Apama Correlator container [here](https://hub.docker.com/_/apama-correlator). More information about building your own image can be found in the parent directory.
```

Comment:
Get the Apama Correlator container [here](https://hub.docker.com/_/apama-correlator). More information about building your own image is contained in the README in the parent directory of this sample.
```suggestion
Get the Apama Correlator container [here](https://hub.docker.com/_/apama-correlator). More information about building your own image is contained in the README in the parent directory of this sample.
```